### PR TITLE
Changed LoadTranscript to be TranscriptLoader and added documentation.

### PR DESCRIPTION
**Pull request by zhg-kj**
I corrected the naming convention problem of LoadTranscript by renaming it as the noun TranscriptLoader. 

I also added documentation for the files:
- TranscriptLoader.java
- TranscriptController.java
- TranscriptService.java

### CODE REVIEW

**Review by Lemiex — APPROVED** [review]
Documentation was helpful! I still think “Transcript” makes the most sense as a name as opposed to TranscriptLoader, but not a big deal so i approved if you want to stick with your naming

**Comment by zhg-kj** [comment]
> 

The issue with simply Transcript is that I think that implies that it is an entity but it really isn’t an entity. It is basically just a model with functions that are used directly on files in the DB

**Comment by zhg-kj** [comment]
> Documentation was helpful! I still think “Transcript” makes the most sense as a name as opposed to TranscriptLoader, but not a big deal so i approved if you want to stick with your naming

The issue with simply Transcript is that I think that implies that it is an entity but it really isn’t an entity. It is basically just a model with functions that are used directly on files in the DB.

File: backend/src/main/java/com/speakfluid/backend/model/TranscriptLoader.java [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
+ * from MongoDB.
+ *
+ * @author  Kai Zhuang
+ * @version 1.0
+ * @since   2022-11-12
+ */
+
+public class TranscriptLoader {
+
+    private String transcriptName;
```
Comment (edwardhuahan) [inline backend/src/main/java/com/speakfluid/backend/model/TranscriptLoader.java:14]: I'm a fan of this naming convention nice

File: backend/src/main/java/com/speakfluid/backend/service/TranscriptService.java [hunk 2]
```diff
@@ -15,6 +15,14 @@
...
+/**
+ * TranscriptService is used to query MongoDB.
+ *
+ * @author  Kai Zhuang
+ * @version 1.0
+ * @since   2022-11-12
+ */
+
 @Service
 public class TranscriptService {
```
Comment (edwardhuahan) [inline backend/src/main/java/com/speakfluid/backend/service/TranscriptService.java:27]: Is there a better name for this? I can't rmr if they call this type of class a Gateway or something

**Review by edwardhuahan — CHANGES_REQUESTED** [review]
Need to fix the merge conflict with Suggestion.java

**Review by edwardhuahan — APPROVED** [review]

**Pull request merged**